packages/graphql-codegen-cli/src/codegen.ts

```diff
@@ -15,8 +15,21 @@ import { getPresetByName } from './presets';
 import { debugLog } from './utils/debugging';
 import { printSchemaWithDirectives } from '@graphql-tools/utils';
 import { CodegenContext, ensureContext } from './config';
+import path from 'path';
+// eslint-disable-next-line
+import { createRequire, createRequireFromPath } from 'module';
 
-export const defaultLoader = (mod: string) => import(mod);
+const makeDefaultLoader = (from: string) => {
+  if (!path.extname(from)) {
+    from = path.join(from, '__fake.js');
+  }
+
+  const relativeRequire = (createRequire || createRequireFromPath)(from);
```

Comment:
What node version is `@graphql-codegen/cli` supposed to support?

Reply:
Node>10 is great, so I think we are fine :)